docs/3.0/recipes/add-count-to-menu-items.md

```diff
@@ -29,7 +29,7 @@ module NavigationLabelWithCount
   end
 
   def resource_count
-    resolve_query_scope.call(model_class: model_class).count
+    index_query.call(model_class: model_class).count
```

Comment:
hmmm not sure if this still work... we should call the execution context

Reply:
What if we remove this recipe from 3.0?

Reply:
Removed